src/lib/utils/meta_tables/meta_chunk_sort_orders_table.cpp

```diff
@@ -27,12 +27,14 @@ std::shared_ptr<Table> MetaChunkSortOrdersTable::_on_generate() const {
   for (const auto& [table_name, table] : Hyrise::get().storage_manager.tables()) {
     for (auto chunk_id = ChunkID{0}; chunk_id < table->chunk_count(); ++chunk_id) {
       const auto& chunk = table->get_chunk(chunk_id);
-      const auto ordered_by = chunk->ordered_by();
+      const auto& ordered_by = chunk->ordered_by();
       if (ordered_by) {
-        std::stringstream order_by_mode_steam;
-        order_by_mode_steam << ordered_by->second;
-        output_table->append({pmr_string{table_name}, static_cast<int32_t>(chunk_id),
-                              static_cast<int32_t>(ordered_by->first), pmr_string{order_by_mode_steam.str()}});
+        for (const auto& ordered_by_single : *ordered_by) {
```

Comment:
Not important (do what you think is best) but you could use structured bindings here to obtain first and second directly.

Reply:
Good suggestion! That helps avoid that awkward variable name.